pages/users.tsx

```diff
@@ -1,5 +1,6 @@
+import React from "react";
 import { inter } from "../styles/fonts";
```

Comment:
This is the second place I am seeing this import, please check all your files that have font imports and move them into your root layout file.

PS. You can set up your next/fonts to use a CSS var, so you can just use that CSS var whenever you need to use that font.